dev_scripts/release-container-image.py

```diff
@@ -0,0 +1,246 @@
+    attest_provenance(full_image, commit, repository, branch)
+    digests = get_platform_digests(full_image)
+    debian_archive_date = extract_debian_archive_date()
+    for platform in ["linux/amd64", "linux/arm64"]:
+        pass  # reproduce_image(full_image, platform, debian_archive_date, digests[platform])
+
+    sign_image(full_image, ghcr_signer_path)
+
+
+def attest_provenance(full_image, commit, repository, branch):
+    click.echo(f"\n🔐 Attesting provenance for image: {full_image}")
+    click.echo(f"   Commit: {commit}\n")
+
+    try:
+        run(
+            [
+                "poetry",
+                "run",
+                "python",
+                "dev_scripts/dangerzone-image",
+                "attest-provenance",
+                "--repository",
+                repository,
+                "--branch",
+                branch,
+                "--commit",
+                commit,
+                full_image,
+            ]
+        )
+        click.echo("\n✅ Provenance attestation successful\n")
+    except subprocess.CalledProcessError as e:
+        click.echo(f"\n❌ Provenance attestation failed: {e}\n")
+        sys.exit(1)
+
+
+def get_candidate_image(commit):
+    short_commit = commit[:7]
+
+    image_name = IMAGE_NAME_PATH.read_text().strip()
+
+    click.echo(f"\n📦 Looking for images for commit: {short_commit}")
+    click.echo(f"   Repository: {image_name}\n")
+
+    # Get the latest container image with this commit
+    try:
+        result = run(
+            [
+                "crane",
+                "ls",
+                "--full-ref",
+                image_name,
+            ],
+            capture_output=True,
+        )
+
+        # Filter images matching the commit
+        images = [line for line in result.stdout.splitlines() if short_commit in line]
+
+        if not images:
+            click.echo(f"❌ No images found for commit {short_commit}")
+            sys.exit(1)
+
+        # Get the latest one
+        latest_image = images[-1]
+
+        # Get the digest
+        result = run(
+            ["crane", "digest", latest_image],
+            capture_output=True,
+        )
+        digest = result.stdout.strip()
+
+        # Format as image@digest
+        image_base = latest_image.split(":")[0]
+        full_image = f"{image_base}@{digest}"
+
+        click.echo(f"✅ Found image:")
+        click.echo(f"   {full_image}\n")
+        return full_image
+
+    except subprocess.CalledProcessError as e:
+        click.echo(f"❌ Error finding candidate image: {e}")
+        sys.exit(1)
+
+
+def get_platform_digests(full_image):
+    """Get platform-specific full_image digests."""
+    click.echo(f"\n📋 Getting platform-specific digests for: {full_image}\n")
+
+    try:
+        result = run(
+            ["crane", "manifest", full_image],
+            capture_output=True,
+        )
+        manifest = json.loads(result.stdout)
+        click.echo("✅ Platform digests retrieved\n")
```

Comment:
Can we perhaps mention here what are the platform digests that we have retrieved?